### PR TITLE
Profile integration for contact point lookup by national identity number

### PR DESCRIPTION
**Pull request by acn-sbuad**
## Description

New logic in integration project to support generation of notifications based of national identity number.
A new profile client has been implemented with methods to check for availability of contact points and to retrieve them. 

## Related Issue(s)
- #437 

## Verification
- [x] **Your** code builds clean without any errors or warnings
- [x] Manual testing done (required)
- [x] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [x] All tests run green

### CODE REVIEW

File: test/Altinn.Notifications.Tests/Notifications.Integrations/Profile/ProfileClientTests.cs [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+        return null;
+    }
+
+    private static object? GetPopulatedListContent<T>()
+    {
+        if (typeof(T) == typeof(UserContactPointAvailabilityList))
+        {
+            var availabilityList = new List<UserContactPointAvailability>
+        {
```
Comment (codemascot) [inline test/Altinn.Notifications.Tests/Notifications.Integrations/Profile/ProfileClientTests.cs:170]: Indentation issue here!

File: test/Altinn.Notifications.Tests/Notifications.Integrations/Profile/ProfileClientTests.cs [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+            new UserContactPointAvailability() { NationalIdentityNumber = "01025101038", EmailRegistered = true },
+            new UserContactPointAvailability() { NationalIdentityNumber = "01025101037", EmailRegistered = false }
+        };
+            return new UserContactPointAvailabilityList() { AvailabilityList = availabilityList };
+        }
+        else if (typeof(T) == typeof(UserContactPointsList))
+        {
+            var contactPointsList = new List<UserContactPoints>
+        {
```
Comment (codemascot) [inline test/Altinn.Notifications.Tests/Notifications.Integrations/Profile/ProfileClientTests.cs:179]: Indentation issue here!

**Review by codemascot — APPROVED** [review]
LGTM

**Review by tba76 — APPROVED** [review]
Kjør på 🍺

**Pull request merged**